src/lwc/rd2ElevateCreditCardForm/rd2ElevateCreditCardForm.js

```diff
@@ -38,31 +43,57 @@ export default class rd2ElevateCreditCardForm extends LightningElement {
         updatePaymentButtonLabel,
         cancelButtonLabel,
         nextPaymentDonationDateMessage,
+        nextACHPaymentDonationDateMessage,
         commonExpirationDate
     };
 
     @track isLoading = true;
     @track isDisabled = false;
     @track alert = {};
 
-    _paymentMethod = undefined;
-    _nextDonationDate = undefined;
+    _paymentMethod;
+    _nextDonationDate;
     _isEditPayment = false;
     
     @api isEditMode;
     @api cardLastFour;
     @api cardLastFourLabel;
     @api cardExpDate;
+    @api achLastFourLabel;
+    @api achLastFour;
+
+    @api payerOrganizationName;
+    @api payerFirstName;
+    @api payerLastName;
+    @api achAccountType;
 
     @api
     get paymentMethod() {
         return this._paymentMethod;
     }
 
     set paymentMethod(value) {
+        this.notifyIframePaymentMethodChanged(value);
         this._paymentMethod = value;
     }
 
+    notifyIframePaymentMethodChanged(newValue) {
+        const iframe = this.selectIframe();
+        if (isNull(iframe)) {
+            return;
```

Comment:
No frame? No Problem. If the frame isn't loaded, it should get the correct value when it loads so we don't need to notify.